helix-core/src/main/java/org/apache/helix/monitoring/mbeans/InstanceMonitor.java

```diff
@@ -61,24 +112,20 @@ public String getSensorName() {
         serializedTags(), _participantName);
   }
 
-  @Override
   public long getOnline() {
-    return _isUp ? 1 : 0;
+    return _onlineStatusGauge.getValue();
   }
 
-  @Override
   public long getEnabled() {
-    return _isEnabled ? 1 : 0;
+    return _enabledStatusGauge.getValue();
   }
 
-  @Override
   public long getTotalMessageReceived() {
-    return _totalMessageReceived;
+    return _totalMessagedReceivedCounter.getValue();
   }
 
-  @Override
   public long getDisabledPartitions() {
-    return _disabledPartitions;
+    return _disabledPartitionsGauge.getValue();
```

Comment:
If no tests are using these methods, we can remove them.
And even that is the case, we can make them protected.